### DISCUSSION TITLE
Lake Shooter

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#151] [Games]
# Lake Shooter

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/lakeshooter/?giscus=44927f5707db3014dd1647a4Vm4abCP8SZIw4YPHaOEDe1JFeJrJw%2BGPfssaD2ZvQdN5%2BgU4pgXpjOZ1FjA1xy4xBUw7aNPEaHJR1coa5Q5ayg%2BAUntVrR7fov67atXSCmBu%2BVjPQ%2Fx9pIkpzqA%3D

**Comment by jimmycartrette** [comment]
My goal was to use this to get proficient coding in C and lower level programming. It's heavily inspired by "River Raid", an Atari 2600 game I liked.
